src/bundle.c

```diff
@@ -553,3 +560,11 @@ gboolean umount_bundle(const gchar *bundlename, GError **error) {
 out:
 	return res;
 }
+
+void free_bundle(RaucBundle *bundle) {
+
+	g_free(bundle->path);
+	g_clear_pointer(&bundle->signer, X509_free);
```

Comment:
There are no signer or verified_chain fields at this point in the patch series.

Reply:
Fixed by reordering patches.